README.md

```diff
@@ -148,6 +174,13 @@ const FancyLink1 = withFontLoading('Lato-Bold')(
 
 This will lazy-load `Lato-Bold` and meanwhile assign a fallback font and styling to the element via `props.$fontStyles`.
 
+```flow
```

Comment:
FYI this strips all syntax highlighting.  `js` at least gives you some, although not for the type annotations.